portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,41 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+//    public ArrayList<String> comments = new ArrayList<String>(List.of("boom", "bam", "pow"));
+
+
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // obtain input from form(the user's comment)
+    String userComment = getParameter(request, "userCommentInput", "");
+    
+    // seperate words into indivdual text
+    String[] words = userComment.split("\\s*,\\s*");
+
+    // Respond with the result.
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println(Arrays.toString(words));
+  }
+  
+//   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+//     // Get the input from the form.
+//     response.setContentType("text/html;");
+//     String userComment = getParameter(request, "userCommentInput", "");
```

Comment:
For a review, please clean up obsolete code.  You can also just copy them elsewhere for now.  Also, they exist in code history. So there is no need to comment them out and makes the code messy.